src/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -611,6 +620,22 @@ public static void GetFullPath_Windows_ValidExtendedPaths(string path)
             }
         }
 
+        [PlatformSpecific(TestPlatforms.Windows)]  // Tests Windows-specific paths
+        [Theory]
+        [InlineData(@"\\.\UNC\")]
+        [InlineData(@"\\.\UNC\server7")]
```

Comment:
Don't ever use anything but LOCALHOST- odds are either someone already has a server name you've chosen or will- particularly on the corpnet- where there are many, many, hosts.